### DISCUSSION TITLE
Options for regridding variables on a CAM-SE grid into lon-lat space

### DISCUSSION THREAD

**Discussion by tlsw231** [pydata/xarray#6768] [Q&A] [answered]
The variables on the unstructured CAM-SE grid in the CESM model are stored with dimensions `(ncol,time)` where `ncol` is a linear index corresponding to all `lat/lon` values. I want to plot some of these in `lon-lat` space. Are there any publicly available resources for doing this? [I believe there might be in-house tools to do this at NCAR.]

**Comment by dcherian** [comment, marked as answer]
I would ask this question here: https://github.com/UXARRAY/uxarray/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

**Comment by tlsw231** [comment]
Thanks! will move it over to that forum.
